prime-router/metadata/hl7_mapping/STLTs/CA/CA.yml

```diff
@@ -4,11 +4,7 @@
 ##########################################################################
 hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
 
-constants:
-  #TODO: Constants are not currently working in sender transforms
-  simpleReportClia: '"36D1332559"'
-  simpleReportName: '"Simple Report"'
-  useAoe: '"false"'
+# TODO: Constants are not currently working in sender transforms
```

Comment:
Since Jimmy's commit merged - might be worth adding the simple report clia/name back and trying to use them?